docs/pages/en-us/Features.md

```diff
@@ -31,9 +31,9 @@ The service is extensively scalable.  Elasticsearch is designed with scalability
 The service is designed for durability as Elasticsearch provides data redundancy across it's highly scalable cluster.
 
 ## High-Availability
-The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.  
+The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.
 
-## Audit Web Service API and Client
-Audit Web Service for logging application audit events through its RESTful interface which sends them through to Elasticsearch.
+## Audit Web Service
```

Comment:
Spoken with @glucy and we have agreed to rename the heading to REST API and to read as: RESTful API which allows audit event messages to be indexed within Elasticsearch.

Committed in the latest commit